data-reconciliation-app/docs/demo-guidance.md

```diff
@@ -2,30 +2,214 @@
 
-## Reference
+## Part 2. Ingestion
+```bash
+  -------- Test Ingestion Service --------  
+
+✅ [Pass]: member0 - CSV data ingest failed (wrong file)
+✅ [Pass]: member0 - CSV data ingest failed (wrong schema)
+✅ [Pass]: member0 - CSV data ingest succeeded
+ ---
+✅ [Pass]: member1 - JSON data ingest failed (data length is zero)
+✅ [Pass]: member1 - JSON data ingest failed (data is null)
+✅ [Pass]: member1 - JSON data ingest succeeded
+ ---
+✅ [Pass]: member2 - JSON data ingest succeeded
+🎬 Ingestion Stage Complete
+```
+
+### Demo notes
+After you press any key, we will start the ingestion service. This service is a simple REST API that accepts JSON/CSV data. You can ingest data for a particular member, and the data will be stored in the network. There are a few negative test cases demonstrated as well as ingesting different file formats. At this point of the demo you will be asked to press any key to continue.
+
+## Part 3. Reporting Service (Full Report)
+```bash
+-------- Test Reporting Service (Full Report) --------  
+
+✅ [Pass]: user0 - Getting report without ingesting data should fail as 'No Data to Report' 
+✅ [Pass]: member0 - Getting all data records should succeed
+✅ [Pass]: member1 - Getting all data records should succeed
+✅ [Pass]: member2 - Getting all data records should succeed
+
+member1 Full Report:
+{
+  "group_status": "LACK_OF_CONSENSUS",
+  "majority_minority": "Majority",
+  "count_of_unique_values": 2,
+  "members_in_agreement": 2,
+  "lei": "984500E1B2CA1D4EKG67",
+  "nace": "A01.1"
+}
+{
+  "group_status": "LACK_OF_CONSENSUS",
+  "majority_minority": "Majority",
+  "count_of_unique_values": 2,
+  "members_in_agreement": 2,
+  "lei": "9845001D460PEJE54159",
+  "nace": "K.65.12"
+}
+{
+  "group_status": "IN_CONSENSUS",
+  "majority_minority": "Majority",
+  "count_of_unique_values": 1,
+  "members_in_agreement": 3,
+  "lei": "254900Z8QM2AR51H5I26",
+  "nace": "C.10.1"
+}
+{
+  "group_status": "IN_CONSENSUS",
+  "majority_minority": "Majority",
+  "count_of_unique_values": 1,
+  "members_in_agreement": 3,
+  "lei": "984500F5BD5BE5767C51",
+  "nace": "C.18.13"
+}
+{
+  "group_status": "LACK_OF_CONSENSUS",
+  "majority_minority": "Majority",
+  "count_of_unique_values": 2,
+  "members_in_agreement": 2,
+  "lei": "9845005E5AEEB78CE366",
+  "nace": "G.46.19"
+}
+{
+  "group_status": "IN_CONSENSUS",
+  "majority_minority": "Majority",
+  "count_of_unique_values": 1,
+  "members_in_agreement": 3,
+  "lei": "984500815D6139D53E23",
+  "nace": "M.69.1"
+}
+{
+  "group_status": "LACK_OF_CONSENSUS",
+  "majority_minority": "Majority",
+  "count_of_unique_values": 2,
+  "members_in_agreement": 2,
+  "lei": "984500UF3DE41EFA7F02",
+  "nace": "B.08.9"
+}
+
+🎬 Full Reports Complete
+
+- Press any key to continue...
+```
+
```

Comment:
nit:

For consistency, `## Demo notes` title is missing

Reply:
Removed Demo notes